parts/xmem_nozzle.go

```diff
@@ -1445,14 +1454,17 @@ func (xmem *XmemNozzle) batchSetMetaWithRetry(batch *dataBatch, numOfRetry int)
 				// lost on conflict resolution on source side
 				// this still counts as data sent
 				additionalInfo := DataFailedCRSourceEventAdditional{Seqno: item.Seqno,
+					VbucketCommon:  VbucketCommon{VBucket: item.Req.VBucket},
 					Opcode:         item.GetMemcachedCommand(),
 					IsExpirySet:    (len(item.Req.Extras) >= 8 && binary.BigEndian.Uint32(item.Req.Extras[4:8]) != 0),
-					VBucket:        item.Req.VBucket,
 					ManifestId:     item.GetManifestId(),
 					Cloned:         item.Cloned,
 					CloneSyncCh:    item.ClonedSyncCh,
 					ImportMutation: item.ImportMutation,
 				}
+				if item.OrigSrcVB != nil {
+					additionalInfo.SetOrigSrcVB(*item.OrigSrcVB)
+				}
 				xmem.RaiseEvent(common.NewEvent(common.DataFailedCRSource, nil, xmem, nil, additionalInfo))
```

Comment:
DerivedData should be passed here.

I ran mobile tests (runHlv 0 and 0a with simulator) and it panics because the DerivedData is expected.

cc: @darshan-na